src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -2489,20 +2490,14 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     ====================================================================================================
   -->
   <Target Name="GenerateBindingRedirectsUpdateAppConfig"
-    AfterTargets="GenerateBindingRedirects"
```

Comment:
Why change the target hook here?

Reply:
The issue is related this target skipped early in the build, but later the condition became true, which then changed the build outputs.  
I remember I had a hard time getting the AfterTargets to get the behavior I wanted, so it was easier to add it to the ResolveReferencesDependsOn property.